src/mempool_limits.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#ifndef BITCOIN_MEMPOOL_LIMITS_H
+#define BITCOIN_MEMPOOL_LIMITS_H
+
+#include <cstddef>
+#include <policy/policy.h>
+
+/**
+ * Options struct containing limit options for a CTxMemPool. Default constructor
+ * populates the struct with sane default values which can be modified.
+ *
+ * Most of the time, this struct should be referenced as CTxMemPool::Limits.
+ */
+struct MemPoolLimits {
+    size_t ancestor_count = DEFAULT_ANCESTOR_LIMIT;
+    size_t ancestor_size_vbytes = DEFAULT_ANCESTOR_SIZE_LIMIT_KVB * 1000;
+    size_t descendant_count = DEFAULT_DESCENDANT_LIMIT;
+    size_t descendant_size_vbytes = DEFAULT_DESCENDANT_SIZE_LIMIT_KVB * 1000;
```

Comment:
Reviewers: What do people think about using `uint64_t` here instead of `size_t`? It seems like there's no obvious need for these to be platform-specific and it might be better to not have to worry about them being inconsistent across platforms.

Reply:
You have a point, but I don't know. All the array/vector size functions return size_t, right? So making these `uint64_t` introduces an implicit cast. That while they're unlikely to ever be outside the range of `uint32_t` anyway.

Reply:
> What do people think about using uint64_t here instead of size_t? It seems like there's no obvious need for these to differ from platform to platform and it might be better to not have to worry about them being inconsistent across platforms.

Yeah makes sense. Do you mean to do those in this PR? It seems there are thoughts about changing all virtual size representation to int64_t or int32_t across the board (signed because we do arithmetic with them). See #23962 and #23957

Reply:
See: https://github.com/bitcoin/bitcoin/pull/25290#issuecomment-1165951176

Reply:
Made these `int64_t`s to avoid an unintended change in behavior. Added a project-wide TODO to have bounds checking in `ApplyArgsManOptions`